addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -24,13 +32,25 @@ const GalleryWidget = publicWidget.Widget.extend({
      * @param {Event} ev
      */
     _onClickImg: function (ev) {
-        if (this.$modal || ev.currentTarget.matches("a > img")) {
+        const clickedEl = ev.currentTarget;
+        if (this.$modal || clickedEl.matches("a > img")) {
             return;
         }
         var self = this;
-        var $cur = $(ev.currentTarget);
 
-        var $images = $cur.closest('.s_image_gallery').find('img');
+        let imageEls = this.el.querySelectorAll("img");
+        const currentImageEl = clickedEl.closest("img");
+        const currentImageIndex = [...imageEls].indexOf(currentImageEl);
+        // We need to reset the images to their original source because it might
+        // have been changed by a mouse event (e.g. "hover effect" animation).
+        let i = 0;
+        imageEls = [...imageEls].map(el => {
+            const cloneEl = el.cloneNode(true);
+            cloneEl.src = this.originalSources[i];
```

Comment:
Nitpick but in the `website.gallery.slideshow` template, we see that we only need the `src` and `alt` image attributes: wouldn't it be better to create a new image element and set these two attributes instead of cloning the whole image with all its attributes, only to use two of them ? :thinking: 
But it could stay like this if you don't want to :wink: (maybe it's better like this, because we won't need to think to add the attributes in an eventual future where this template would change)

Reply:
yes, I prefer to leave it as it has always been for now ;)